### PR TITLE
Added db functionality

### LINKED ISSUE #14: Implement new database functionality
- [x] Get by Student ID, return Learning Style, on endpoint /students/<student_id>/learningstyle

- [x] Get all Resources filtered to only their ID's and LearningStyle on endpoint /resources/learningstyles

### PR DESCRIPTION
**Pull request by DevenMarrero**
#14

### CODE REVIEW

File: api/controller/endpoints.py [hunk 1]
```diff
@@ -15,13 +15,16 @@
 #  You should have received a copy of the GNU Affero General Public License
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+from bson.objectid import ObjectId
```
Comment (jerryxu99) [inline api/controller/endpoints.py]: looks unused

**Review by jerryxu99 — CHANGES_REQUESTED** [review]
i think the bson import is being unused. I imagine you were using it to try different ways of querying by id but should be removed now

File: api/controller/endpoints.py (hunk 1 above)
Comment (DevenMarrero) [reply api/controller/endpoints.py]: Can needs to be removed, I think it was automatically imported by vscode when I was testing some stuff

Also need to double check that the student api is searching for student_id and not "BAZ"

**Review by varun-patel — APPROVED** [review]
@DevenMarrero I will approve your changes and merge when ready.

**Pull request merged**